### ISSUE TITLE
can i finetune the model ?

### ISSUE THREAD

**Issue by boscoj2008** [JohnGiorgi/DeCLUTR#249]
I wondering if there is a way to fine-tune on a domain task rather than training on my small data set. Thanks

**Comment by JohnGiorgi** [comment]
Yup. The model is accessible via [Transformers](https://huggingface.co/johngiorgi/declutr-base). So you can load it in that library and fine-tune it on your task in PyTorch or Tensorflow. There is documentation [here](https://huggingface.co/docs/transformers/training). Just change the pretrained model name to `"johngiorgi/declutr-base"` (or our other pretrained models, see [here](https://huggingface.co/johngiorgi) for a list).

**Comment by JohnGiorgi** [comment]
Closing, please feel free to re-open if your question is not answered. Note that you can also load our models in the [SentenceTransformers](https://www.sbert.net/index.html) library (I've added some documentation on that [here](https://github.com/JohnGiorgi/DeCLUTR#embedding)), so you can take full advantage of that library's utilities for training.
